CODEOWNERS

```diff
@@ -96,6 +98,7 @@
 /gallery/workbook/microsoft.network* @microsoft/network-insights @microsoft/applicationinsights-devtools
 /gallery/workbook/microsoft.wvd-insights* @microsoft/windows-virtual-desktop
 /gallery/adxcluster-insights/microsoft.kusto-clusters.json @microsoft/kusto-workbooks
+/gallery/mysqlflexibleserver-insights/** @microsoft/azuremysql
```

Comment:
you didn't update this along with the move of this file to `/gallery/workbook/microsoft.dbformysql-flexibleservers.json`?
right now this line says you own a folder that doesn't exist instead of the file that does?

Reply:
Has been updated